docs/guides/fetch-data/read-profile-data.md

```diff
@@ -271,39 +275,37 @@ fetchProfileData();
 
 ```javascript title="read_profile.js"
 // ...
-// Fetchable picture information
-let baseURL = 'https://ipfs.lukso.network/ipfs/';
 
+// Fetchable picture information
 let backgroundImageLinks = [];
-let fullSizeBackgroundImg;
-
 let profileImageLinks = [];
+let fullSizeBackgroundImg;
 let fullSizeProfileImg;
 
 /*
- * Fetch picture information from the JSON dataset of
- * a Universal Profile
+ * Fetch the Universal Profile and extract the 
+ * picture information from its JSON dataset
  *
  * @return string Error
  */
-async function fetchPictureData() {
-  pictureData = await getProfile(SAMPLE_PROFILE_ADDRESS);
+async function fetchProfilePictures() {
+  pictureData = await getProfileData(SAMPLE_PROFILE_ADDRESS);
   let backgroundImagesIPFS =
-    pictureData.LSP3Profile.LSP3Profile.backgroundImage;
-  let profileImagesIPFS = pictureData.LSP3Profile.LSP3Profile.profileImage;
+    pictureData.value.LSP3Profile.backgroundImage;
+  let profileImagesIPFS = pictureData.value.LSP3Profile.profileImage;
 
   try {
     for (let i in backgroundImagesIPFS) {
       backgroundImageLinks.push([
         i,
-        baseURL + backgroundImagesIPFS[i].url.substring(7),
+        IPFS_GATEWAY + backgroundImagesIPFS[i].url.substring(7),
```

Comment:
I think it is easier to understand if you do this instead:

```suggestion
       backgroundImagesIPFS[i].replace('ipfs://', IPFS_GATEWAY)
```

It is also safer as nothing guarantee, with your code, that the URLs starts with ipfs:// - what if a user simply puts `https://google.com/image.png` ?